study.md

```diff
@@ -34,7 +34,18 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+The map method gives us a new array with the result of the applied function to
+the original array.
+The map enumerator gives us a new array after running a block for every elemenyt, so my udnerstanding
+is that it is mainly a syntax difference.
+
+Each is another method that makes the differences stand out to me.
+The enumerable model does not define the each method, the module does so,
```

Comment:
`each` is defined in the `Enumerable` module.  `Array` gets its `each` method from `Enumerable` because it's included.

The question about why `Array` might redefine methods from `Enumerable` was not answered.

`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.